File: crypto/src/main/java/io/oversec/one/crypto/LoggingConfig.java
package io.oversec.one.crypto;

import android.util.Log;
import roboguice.util.Ln;

public class LoggingConfig {
    public static boolean LOG = false;

    public static void init(boolean debug) {
            setLog(debug);
    }

    public static void setLog(boolean b) {
        LOG = b;
        if (!LOG) {
            Ln.getConfig().setLoggingLevel(Log.ERROR);
        } else {
            Ln.getConfig().setLoggingLevel(Log.VERBOSE);
        }
    }
}

File: crypto/src/main/java/io/oversec/one/crypto/LoggingConfig.kt
package io.oversec.one.crypto

import android.util.Log
import roboguice.util.Ln

object LoggingConfig {
    var LOG = false

    fun init(debug: Boolean) {
        setLog(debug)
    }

    fun setLog(b: Boolean) {
        LOG = b
        if (!LOG) {
            Ln.getConfig().loggingLevel = Log.ERROR
        } else {
            Ln.getConfig().loggingLevel = Log.VERBOSE
        }
    }
}
